pulp_deb/app/tasks/publishing.py

```diff
@@ -237,7 +237,10 @@ def __init__(
         if suite:
             self.release["Suite"] = suite
         self.release["Version"] = version
-        self.release["Codename"] = codename
+        if codename:
+            self.release["Codename"] = codename
+        else:
+            self.release["Codename"] = "fallback"
```

Comment:
```suggestion
        self.release["Codename"] = codename or "fallback"
```
Also why not fallback to the distribution?

Reply:
Allready changed it from "fallback" to the base of the distribution (`distribution.split("/")[0]`), but I did not see your more concise  syntax in time, will amend.